zig nightly: .One -> .one

```sh
~/gh-zig/websocket.zig on master
1 % zig build test
test
└─ run test
   └─ zig test Debug native 1 errors
src/client/client.zig:25:18: error: no field named 'One' in enum 'builtin.Type.Pointer.Size'
                .One => return ReadLoopHandler(ptr_info.child),
                ~^~~
/Users/erg/gh/zig/build/stage3/lib/zig/std/builtin.zig:623:26: note: enum declared here
        pub const Size = enum(u2) {
                         ^~~~
src/client/client.zig:162:40: note: called from here
        const Handler = ReadLoopHandler(@TypeOf(handler));
                        ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
error: the following command failed with 1 compilation errors:
/Users/erg/gh/zig/build/stage3/bin/zig test -ODebug --dep build -Mroot=/Users/erg/gh-zig/websocket.zig/src/websocket.zig -Mbuild=/Users/erg/gh-zig/websocket.zig/.zig-cache/c/9e1376e0f9f98a38989a65451965fe75/options.zig -lc --test-runner /Users/erg/gh-zig/websocket.zig/test_runner.zig --cache-dir /Users/erg/gh-zig/websocket.zig/.zig-cache --global-cache-dir /Users/erg/.cache/zig --name test --zig-lib-dir /Users/erg/gh/zig/build/stage3/lib/zig/ --listen=-
Build Summary: 1/4 steps succeeded; 1 failed
test transitive failure
└─ run test transitive failure
   └─ zig test Debug native 1 errors
error: the following build command failed with exit code 1:
/Users/erg/gh-zig/websocket.zig/.zig-cache/o/a1fe7512ae89e138a6f320035c67090c/build /Users/erg/gh/zig/build/stage3/bin/zig /Users/erg/gh/zig/build/stage3/lib/zig /Users/erg/gh-zig/websocket.zig /Users/erg/gh-zig/websocket.zig/.zig-cache /Users/erg/.cache/zig --seed 0x5c4ae3b8 -Z1a489a107a7e07c2 test

~/gh-zig/websocket.zig on master
```

--- a/src/client/client.zig
+++ b/src/client/client.zig
@@ -22,7 +22,7 @@ fn ReadLoopHandler(comptime T: type) type {
         },
         .pointer => |ptr_info| {
             switch (ptr_info.size) {
-                .One => return ReadLoopHandler(ptr_info.child),
+                .one => return ReadLoopHandler(ptr_info.child),
                 else => @compileError("readLoop: handler does not support Slice, C and Many pointers."),
             }
         },

